repos/c2sm/packages/icon/package.py

```diff
@@ -596,7 +596,6 @@ def configure_args(self):
                 config_vars['LOC_ICON4PY_INTERPOLATION'].append(
                     self.spec['py-icon4py:interpolation'].headers.
                     directories[0])
-            if self.spec['py-icon4py'].version > Version("0.0.8"):
```

Comment:
Wouldn't you need to set that to `Version("0.0.7)`?
Otherwise this is applied also for older versions of icon4py